src/test/java/parser/ParserTest.java

```diff
@@ -38,30 +44,30 @@ public void getUserCommand_viewAuthors_parsedCorrectly() {
 
     @Test
     public void getUserCommand_viewMangas_parsedCorrectly() {
-        parseInputAssertCommandType("view -a \"test\"", ViewMangasCommand.class);
+        parseInputAssertCommandType("view -a test", ViewMangasCommand.class);
     }
 
     //@@author sarahchow03
     @Test
     public void getUserCommand_deleteAuthor_parsedCorrectly() {
-        parseInputAssertCommandType("catalog -d -a \"test\"", DeleteAuthorCommand.class);
+        parseInputAssertCommandType("catalog -a test -d", DeleteAuthorCommand.class);
     }
 
     @Test
     public void getUserCommand_deleteManga_parsedCorrectly() {
-        parseInputAssertCommandType("catalog -d -a \"test\" -m \"test\"", DeleteMangaCommand.class);
+        parseInputAssertCommandType("catalog -a test -m test -d", DeleteMangaCommand.class);
```

Comment:
Caught this issue in the most recent implementation as well. Good job on the quick fix!